libflux/src/flux/semantic/builtins.rs

```diff
@@ -0,0 +1,338 @@
+                "erfcinv" => Node::Builtin("forall [] (x: float) -> float"),
+                "erfinv" => Node::Builtin("forall [] (x: float) -> float"),
+                "exp" => Node::Builtin("forall [] (x: float) -> float"),
+                "exp2" => Node::Builtin("forall [] (x: float) -> float"),
+                "expm1" => Node::Builtin("forall [] (x: float) -> float"),
+                "floor" => Node::Builtin("forall [] (x: float) -> float"),
+                "gamma" => Node::Builtin("forall [] (x: float) -> float"),
+                "hypot" => Node::Builtin("forall [] (x: float, y: float) -> float"),
+                "j0" => Node::Builtin("forall [] (x: float) -> float"),
+                "j1" => Node::Builtin("forall [] (x: float) -> float"),
+                "log" => Node::Builtin("forall [] (x: float) -> float"),
+                "log10" => Node::Builtin("forall [] (x: float) -> float"),
+                "log1p" => Node::Builtin("forall [] (x: float) -> float"),
+                "log2" => Node::Builtin("forall [] (x: float) -> float"),
+                "logb" => Node::Builtin("forall [] (x: float) -> float"),
+                "mMax" => Node::Builtin("forall [] (x: float, y: float) -> float"),
+                "mMin" => Node::Builtin("forall [] (x: float, y: float) -> float"),
+                "mod" => Node::Builtin("forall [] (x: float, y: float) -> float"),
+                "nextafter" => Node::Builtin("forall [] (x: float, y: float) -> float"),
+                "pow" => Node::Builtin("forall [] (x: float, y: float) -> float"),
+                "remainder" => Node::Builtin("forall [] (x: float, y: float) -> float"),
+                "round" => Node::Builtin("forall [] (x: float) -> float"),
+                "roundtoeven" => Node::Builtin("forall [] (x: float) -> float"),
+                "sin" => Node::Builtin("forall [] (x: float) -> float"),
+                "sinh" => Node::Builtin("forall [] (x: float) -> float"),
+                "sqrt" => Node::Builtin("forall [] (x: float) -> float"),
+                "tan" => Node::Builtin("forall [] (x: float) -> float"),
+                "tanh" => Node::Builtin("forall [] (x: float) -> float"),
+                "trunc" => Node::Builtin("forall [] (x: float) -> float"),
+                "y0" => Node::Builtin("forall [] (x: float) -> float"),
+                "y1" => Node::Builtin("forall [] (x: float) -> float"),
+
+                "float64bits" => Node::Builtin("forall [] (f: float) -> uint"),
+                "float64frombits" => Node::Builtin("forall [] (b: uint) -> float"),
+                "ilogb" => Node::Builtin("forall [] (x: float) -> int"),
+                "frexp" => Node::Builtin("forall [] (f: float) -> {frac: float | exp: int}"),
+                "lgamma" => Node::Builtin("forall [] (x: float) -> {lgamma: float | sign: int}"),
+                "modf" => Node::Builtin(r#"forall [] (f: float) -> {"int": float | frac: float}"#),
```

Comment:
Would it be possible from context to parse this without the quotes? Meaning if we changed the lexer to not treat type names as keywords but as identifiers and then the parser can understand when an identifier is a type name or not? 

This works for now, but when we go to incorporate this type language into Flux itself it would be good if type names were not parsed as keywords.

Reply:
It might mean removing the distinction between keywords and identifiers in the type expression grammar.  If we ever allow users to name their own types in Flux, then we would probably need to remove this distinction anyway.